ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -445,10 +444,11 @@ def update_subscription(self, channel):
             subscription_name = constants.ODF_SUBSCRIPTION
         else:
             subscription_name = constants.OCS_SUBSCRIPTION
+        provider_cluster = config.ENV_DATA["cluster_type"].lower() == "provider"
         kind_name = (
             "subscription.operators.coreos.com"
             if config.multicluster
-            else "subscription"
+            else "subs" if provider_cluster else "subscription"
```

Comment:
please use elif instead of ternary operator `"else 'subs' if provider_cluster else "subscription""`
It should be something like
```
if config.multicluster:
    kind_name = "subscription.operators.coreos.com"
elif provider_cluster:
    kind_name = "subs"
else:
    kind_name = "subscription"
```

Reply:
Addressed.